.gitmodules

```diff
@@ -13,9 +13,6 @@
 [submodule "external/bzip2"]
 	path = external/bzip2
 	url = git://sourceware.org/git/bzip2.git
-[submodule "external/boost"]
-	path = external/boost
-	url = https://github.com/boostorg/boost.git
```

Comment:
`freetype2-testing` is thought of as a "mono testing repo" with (potentially) different sub projects (even though it only does fuzzing right now). Hence, the fuzzing code lives in `/fuzzing/src` instead of in `/src`. Just checking in to make sure: if we _were_ to add the boost submodule back at any point in the future (for a different project) that wouldn't break things on your end, right?

Also - we should mention running the fuzzer as part of Chromium (+ link https://source.chromium.org/chromium/chromium/src/+/master:third_party/?q=third_party) somewhere in here for future reference: https://github.com/freetype/freetype2-testing/blob/master/fuzzing/README.md

Reply:
> Just checking in to make sure: if we _were_ to add the boost submodule back at any point in the future (for a different project) that wouldn't break things on your end, right?

No it would not pose a problem to add it as a git external subproject or use it in files which are not fuzzing related.

> Also - we should mention running the fuzzer as part of Chromium (+ link https://source.chromium.org/chromium/chromium/src/+/master:third_party/?q=third_party) somewhere in here for future reference: https://github.com/freetype/freetype2-testing/blob/master/fuzzing/README.md

At the moment, we're exploring that and the PRs I am doing here are in preparation for that. If it's okay with you, I'll add such a note once it's staged and actually running. Tracking bug is here: https://bugs.chromium.org/p/chromium/issues/detail?id=1140584

Reply:
Brilliant, sounds perfect to me 👍